client/src/app/site/projector/components/presentation-control/presentation-control.component.html

```diff
@@ -1,40 +1,40 @@
-<mat-expansion-panel *ngIf="elements.length">
+<mat-expansion-panel *ngIf="(projections | async) as value">
     <mat-expansion-panel-header>
         <span>{{ 'Media file' | translate }}</span>
     </mat-expansion-panel-header>
-    <div *ngFor="let element of elements">
-        <i>{{ getMediafile(element).getTitle() }}</i>
-        <div *ngIf="getMediafile(element).isImage()">
-            <button type="button" *ngIf="!element.fullscreen" mat-icon-button (click)="fullscreen(element)">
+    <div *ngFor="let projection of value">
```

Comment:
I have committed an update. In my opinion the problem is the "double" async-pipe. The first could be resolved, the second not sufficient. With my commit the result of the async-pipe is stored and then used.